audit/projects/k8s-staging-kubetest2/description.json

```diff
@@ -0,0 +1,11 @@
+{
+  "createTime": "2021-03-24T18:14:58.836Z",
+  "lifecycleState": "ACTIVE",
+  "name": "k8s-staging-kubetest2",
```

Comment:
All files under projects/k8s-staging-kubetest2 are expected, a result of https://github.com/kubernetes/k8s.io/pull/1819 merging